public/src/components/edit_account.jsx

```diff
@@ -72,7 +74,12 @@ class EditAccountInfo extends Component {
           label="Phone Number"
           component={this.renderField}
         />
-        <button type="submit">Update</button>
+        <ButtonToolbar>
+          <Button bsStyle="primary" type="submit">Update</Button>
+          <Link className="btn btn-default" to={`/account/${id}`}>
```

Comment:
I think technically we could make a click handler that uses the history property from this.props and push. Not sure what best practices are. But looks good.